web/src/reducers/auth.js

```diff
@@ -11,19 +11,30 @@ const initialState = {
   initialCheck: false,
   fetching: false,
   authenticated: false,
-  error: ''
+  error: '',
+  user: null
```

Comment:
Interestingly, we've never stored the logged-in user's information anywhere.  Surprising!